Makefile

```diff
@@ -228,43 +214,21 @@ else
 endif
 
 compile_lib: config qemu/config-host.h-timestamp
-	cd qemu && $(MAKE) -j 4
-	$(MAKE) unicorn
+	$(MAKE) -C qemu -j 4
 
-unicorn: $(LIBRARY) $(ARCHIVE)
+unicorn: compile_lib $(LIBRARY) $(ARCHIVE)
 
-$(LIBRARY): $(UC_TARGET_OBJ) uc.o list.o
-ifeq ($(UNICORN_SHARED),yes)
-ifeq ($(V),0)
```

Comment:
Cmake also handles verbosity. But im not a cmake or autotools fan at all. If you do custom makefiles and just want a configure step you may try acr instead. Its equally portable and way smaller and faster